### DISCUSSION TITLE
Dashboard accounts order

### DISCUSSION THREAD

**Discussion by tiagodj** [firefly-iii/firefly-iii#7888] [General]
Hi,

Is it possible to customize the order the accounts appear in the dashboard/home screen?

I can't seem to understand what order is that. It's not by balance, alphabetical, by last activity, or by the Asset accounts order under `accounts/asset`.

I looked for info in the docs and in the UI but couldn't figure it out.

Thanks,
Marcos

**Comment by JC5** [comment]
They are currently ordered by ID but I'm not sure. I'll check it out

**Comment by JC5** [comment]
Thanks for pointing this out. They will be ordered by the "order" from now on.

**Comment by tiagodj** [comment]
awesome, thanks!
